Match sub_0801E99C

Catobat opened the pull request.

hatal175 commented:
You can remove the FORCE_REGISTER if you move the variable definition one row down:
https://nonmatch.netlify.app/functions/122/submissions/3577

Catobat commented:
Nice, thanks! Updated.
I've definitely tried that before. Maybe it was before I simplified the switch and if blocks, and that affected the priorities for which variable deserves the r7 spot.

notyourav commented:
https://github.com/zeldaret/tmc/actions/runs/3832933176/jobs/6526164896#step:4:1

Please make these formatting fixes
